libsodium/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ set(SRC libsodium/src/libsodium)
 set(srcs
     "${SRC}/crypto_aead/chacha20poly1305/aead_chacha20poly1305.c"
     "${SRC}/crypto_aead/xchacha20poly1305/aead_xchacha20poly1305.c"
+    "${SRC}/crypto_aead/aegis256/aead_aegis256.c"
+    "${SRC}/crypto_aead/aegis256/aegis256_soft.c"
+    "${SRC}/crypto_aead/aegis128l/aead_aegis128l.c"
+    "${SRC}/crypto_aead/aegis128l/aegis128l_soft.c"
```

Comment:
Could you please also add related test cases, i.e. extend
TEST_CASES with aead_aegis128l.c and aead_aegis256.c
https://github.com/espressif/idf-extra-components/blob/6e71b67377eaa8d00c02fcfcf42948e36c2f669d/libsodium/test_apps/main/CMakeLists.txt#L3

and them call them in
https://github.com/espressif/idf-extra-components/blob/6e71b67377eaa8d00c02fcfcf42948e36c2f669d/libsodium/test_apps/main/test_sodium.c#L32-L39

Reply:
Added tests now!